src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -158,7 +158,7 @@ or PlaybackStateCompat.StateSkippingToQueueItem
 			}
 			catch (Exception e)
 			{
-				System.Diagnostics.Trace.WriteLine($"[error] {e}, {e.Message}");
+				System.Diagnostics.Trace.TraceError($"[error] {0}", e);
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.